Pluses Server/src/ru/shemplo/pluses/logic/HelpHandler.java

```diff
@@ -0,0 +1,46 @@
+package ru.shemplo.pluses.logic;
+
+import static ru.shemplo.pluses.network.message.AppMessage.MessageDirection.*;
+import static ru.shemplo.pluses.network.message.ControlMessage.ControlType.*;
+import static ru.shemplo.pluses.Run.*;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+import ru.shemplo.pluses.network.message.ControlMessage;
+import ru.shemplo.pluses.network.message.Message;
+import ru.shemplo.pluses.network.pool.AppConnection;
+
+
+public class HelpHandler {
+	static private Long lastUpdated = 0L;
```

Comment:
make it at least volatile (situation when several users send requests)
more appropriate way AtomicLong